docs/docsite/rst/playbooks_lookups.rst

```diff
@@ -401,7 +406,9 @@ sort                         no          list      [pymongo default]      Sortin
 
 Please check https://api.mongodb.org/python/current/api/pymongo/collection.html?highlight=find#pymongo.collection.Collection.find for more detais.
 
-Since there are too many parameters for this lookup method, below is a sample playbook which shows its usage and a nice way to feed the parameters::
+Since there are too many parameters for this lookup method, below is a sample playbook which shows its usage and a nice way to feed the parameters:
+
+.. code-block:: yaml
```

Comment:
YAML+Jinja whould be the default so we shouldn't need this.  If we do need to be explicit, does it take 
```
.. code-block:: YAML+Jinja
```

so that the jinja portions get highlighted?